lib/publiccloud/provider.pm

```diff
@@ -633,7 +633,7 @@ This method is called called after each test on failure or success.
 =cut
 sub cleanup {
     my ($self) = @_;
-    $self->terraform_destroy();
+    $self->terraform_destroy() unless script_run('test -d ' . TERRAFORM_DIR . '/.terraform');
```

Comment:
Why is this needed? It breaks the HA/SAP cleanup....

Reply:
we understood that if there is not terraform directory the terraform_destroy command will fail

Reply:
proposed solution https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13328